src/harness/testcases/testdata/fss_record_c.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I think you can just name this "fss_record.json" -- no need for the "_c" (same for the GWPZ file below).

Reply:
Sorry, my original suggestion was slightly flawed -- I should have suggested "fss_record_0.json" (so that we can create more in the future, if needed; this will match what we've done for similar data files).